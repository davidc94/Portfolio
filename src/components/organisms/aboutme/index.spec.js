import React from 'react';
import renderer from 'react-test-renderer';
import AboutMe from './index';

describe('AboutMe component', () => {
  it('render AboutMe as expected', () => {
    const render = renderer.create(
      <AboutMe />
    );
    expect(render.toJSON()).toMatchSnapshot();
  });
});