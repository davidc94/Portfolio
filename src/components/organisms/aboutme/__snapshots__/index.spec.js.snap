// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutMe component render AboutMe as expected 1`] = `
<div
  className="sc-htpNat dUGsjj"
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flexDirection": "column",
      "justifyContent": "flex-start",
      "padding": "48px 0px",
    }
  }
  width="full"
>
  <div
    className="sc-htpNat dUGsjj"
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "row",
        "justifyContent": "flex-start",
      }
    }
    width="full"
  >
    <div
      className="sc-htpNat kHdJfM"
      style={
        Object {
          "flex": 1,
        }
      }
    />
    <div
      className="sc-htpNat kHdJfM"
      style={
        Object {
          "flex": 2,
        }
      }
    >
      <h1
        className="sc-bwzfXH dtrOTQ"
        color="#2B303A"
        size="large"
      >
        About Me
      </h1>
    </div>
    <div
      className="sc-htpNat kHdJfM"
      style={
        Object {
          "flex": 1,
        }
      }
    />
  </div>
  <div
    className="sc-htpNat dUGsjj"
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "row",
        "justifyContent": "flex-start",
        "maxWidth": "1080px",
      }
    }
    width="full"
  >
    <div
      className="sc-htpNat kHdJfM"
      style={
        Object {
          "flex": 1,
        }
      }
    />
    <div
      className="sc-htpNat kHdJfM"
      style={
        Object {
          "flex": 4,
          "flexDirection": "column",
        }
      }
    >
      <p
        className="sc-bdVaJa iKrFbH"
        size="large"
      >
        Hello, my name is David. I'm a 24 year old developer from London who graduated from Brunel University in 2017 with a Bachelors degree in Computer Science. I've been working at giffgaff as a software developer since then.
      </p>
    </div>
    <div
      className="sc-htpNat kHdJfM"
      style={
        Object {
          "flex": 1,
        }
      }
    />
  </div>
</div>
`;
